tutorials/demo.rst

```diff
@@ -46,7 +48,8 @@ This firmware can help you quickly get started and test
 the core features of OpenWISP Cloud.
 
 If you prefer to use your existing firmware,
-please refer to the :ref:`the alternative firmware instructions
+please refer to the alternative instructions by reading
+:ref:`the alternative firmware instructions
```

Comment:
I am not super convinced of this changed proposed by the copywriter, we have other parts of the text where we link the alternative firmware instructions directly, so this would be inconsistent, moreover, what's the point of repeating it?

Let me know what you think, maybe we can rollback this change.